### PR TITLE
Convert default/scripting/techs/ship_hulls/SHP_XENTRONIUM_HULL.focs

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: parse/EnumPythonParser.cpp [hunk 1]
```diff
@@ -84,5 +84,16 @@ void RegisterGlobalsEnums(boost::python::dict& globals) {
         globals[op.first] = enum_wrapper<PlanetType>(op.second);
     }
 
+    for (const auto& op : std::initializer_list<std::pair<const char*, UnlockableItemType>>{
+            {"Building",   UnlockableItemType::UIT_BUILDING},
+            {"ShipPart",   UnlockableItemType::UIT_SHIP_PART},
+            {"ShipHull",   UnlockableItemType::UIT_SHIP_HULL},
+            {"ShipDesign", UnlockableItemType::UIT_SHIP_DESIGN},
+            {"UnlockTech", UnlockableItemType::UIT_TECH},
```
Comment (geoffthemedio) [inline parse/EnumPythonParser.cpp]: having `Unlock` in the token for just techs is awkward

Comment (o01eg) [reply parse/EnumPythonParser.cpp]: But just `Tech` token already used to declare techs itself.

Comment (geoffthemedio) [reply parse/EnumPythonParser.cpp]: makes sense, but why are all the other tokens not also already used the same way?

Comment (o01eg) [reply parse/EnumPythonParser.cpp]: Sounds reasonable for future conversion.

Comment (o01eg) [reply parse/EnumPythonParser.cpp]: Changed it everywhere.

**Comment by geoffthemedio** [comment]
otherwise seems fine

**Pull request merged**